### PR TITLE
13284 tooltip on share button

### PR DESCRIPTION
**Pull request by flxschmidt**
## Description
Shows a Tooltip over new Export-Certificates BarButton, when navigate to Certificates Overview.

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-13284

## Screenshots
Screen Shot 2022-06-09 at 11 36 23

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Views/TooltipViewController.swift [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+		enaLabel.textColor = .enaColor(for: .darkBackground)
+		enaLabel.numberOfLines = 0
+		enaLabel.lineBreakMode = .byWordWrapping
+		enaLabel.translatesAutoresizingMaskIntoConstraints = false
+		return enaLabel
+	}()
+
+	let viewModel: TooltipViewModel
+
+	init(
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/TooltipViewController.swift:58]: Missing mark

File: src/xcode/ENA/ENA/Source/Views/TooltipViewController.swift [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+		enaLabel.style = .body
+		enaLabel.textColor = .enaColor(for: .darkBackground)
+		enaLabel.numberOfLines = 0
+		enaLabel.lineBreakMode = .byWordWrapping
+		enaLabel.translatesAutoresizingMaskIntoConstraints = false
+		return enaLabel
+	}()
+
+	let viewModel: TooltipViewModel
+
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/TooltipViewController.swift:57]: Missing Mark

File: src/xcode/ENA/ENA/Source/Views/TooltipViewController.swift [hunk 3]
```diff
@@ -0,0 +1,136 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class TooltipViewController: UIViewController, UIPopoverPresentationControllerDelegate {
+
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/TooltipViewController.swift:8]: Missing mark, please move it below init, for reference please see other VCs

File: src/xcode/ENA/ENA/Source/Views/TooltipViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,36 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+class TooltipViewModel {
+	var title: String {
```
Comment (naveeddotio) [inline src/xcode/ENA/ENA/Source/Views/TooltipViewModel.swift:8]: Missing Marks and please reorder

**Review by naveeddotio — CHANGES_REQUESTED** [review]
Looks good, only some small things.

**Review by naveeddotio — APPROVED** [review]

**Pull request merged**